mlx/utils.cpp

```diff
@@ -329,4 +329,13 @@ std::ostream& operator<<(std::ostream& os, const std::vector<size_t>& v) {
   return os;
 }
 
+std::ostream& operator<<(std::ostream& os, const std::vector<int64_t>& v) {
```

Comment:
Probably time to templatize these.. though I'm not really sure we need this function here.. I guess you added it for debugging?

Reply:
Yes, I was printing them to make sure things went around correctly